src/server/editorServices.ts

```diff
@@ -1333,7 +1333,7 @@ namespace ts.server {
                 configHasFilesProperty: parsedCommandLine.raw.files !== undefined,
                 configHasIncludeProperty: parsedCommandLine.raw.include !== undefined,
                 configHasExcludeProperty: parsedCommandLine.raw.exclude !== undefined,
-                wildcardDirectories: createMapFromTemplate(parsedCommandLine.wildcardDirectories),
+                wildcardDirectories: createMapFromTemplate(parsedCommandLine.wildcardDirectories!), // TODO: GH#18217

```

Comment:
That function should work for undefined inputs.

Reply:
We do not allow undefined in `for (... in x)`. we should.